guides/source/debugging_rails_applications.md

```diff
@@ -236,6 +236,11 @@ Below each database statement you can see arrows pointing to the specific source
 
 Verbose query logs are enabled by default in the development environment logs after Rails 5.2.
 
+```rb
+# ./config/environments/development.rb
```

Comment:
I think this is redundant to the statement above:

> After running `ActiveRecord.verbose_query_logs = true` in the `bin/rails console` session to enable verbose query logs and running the method again, it becomes obvious what single line of code is generating all these discrete database calls:

Although they are changing the setting in different places, I don't see the point of having both. I would prefer to mention setting the flag in the config file rather than in a console.

Reply:
> Although they are changing the setting in different places, I don't see the point of having both. I would prefer to mention setting the flag in the config file rather than in a console.

Agreed: the console version is stated above quite clearly. But I didn't want to make the change in the console, I wanted to change the `development.rb` file. I had to go search out the config setting specifically, which was a minor annoyance, and the genesis for this PR.

The point you have made above seems uniform with the rest of the docs: 

![uniformity](https://github.com/user-attachments/assets/466a6130-6d4c-43bc-a504-bc06141db28a)

However, my own 2 cents: to leave the console settings, and _add_ the environmental settings. 

Please confirm whether this is the change you want?

```diff
- After running ActiveRecord.verbose_query_logs = true in the bin/rails console session to enable 
- verbose query logs and running the method again, it becomes obvious what single line of code is 
- generating all these discrete database calls:
+ # ./config/environments/development.rb
+ config.active_record.verbose_query_logs = true
```

Reply:
All of the settings you highlighted are enabled by default in development for newly-generated Rails applications.

Maybe your app predates that, or we (sometimes do) change the defaults in that file.

I think something like "this is enabled by default for new Rails apps in development, but you can enable it by adding this to your config:" with a code block for the flag.

We have different wording there in each section that could all be cleaned up. I will let you decide how to phrase it.

I don't think we need to show the explicit config environment file path, but it might be useful to have both "use `ActiveRecord.whatever_class_variable = true` or via `config.active_record.whatever_thing = true`.

Sorry to sign you up for extra work, but while you're in that file would you mind tidying things up a bit? There might be other cases like that and I'd rather we make them consistent in a single pass. If you're interested, of course.

Reply:
> Sorry to sign you up for extra work, but while you're in that file would you mind tidying things up a bit? There might be other cases like that and I'd rather we make them consistent in a single pass. If you're interested, of 

It is my pleasure: ye have asked and ye shall receive.

I have liberally updated (hopefully) accordance with your suggestions:

1. Made concise where possible.
2. Applied file locations within markup as opposed to within prose where possible due to the above.
3. I did not see much benefit in toggling `ActiveJob.verbose_enqueue_logs = true` within the irb, but i have provided it nonetheless.
4. In terms of doing the same for the [`query_log_tags `](https://api.rubyonrails.org/classes/ActiveRecord/QueryLogs.html). I have skipped it entirely.
5. `in Initializer section` i have assumed this to mean `config/initializers`, perhaps it may refer to environment initializers.

Any issues pls LMK.